task-3/src/index.ts

```diff
@@ -21,6 +22,7 @@ app.use(cors());
 app.use(express.json());
 app.use("/api/users", usersRouter);
 app.use("/api/groups", groupsRouter);
+app.use("/api/login", authRouter);
 app.use(notFoundHandler);
 app.use(errorHandler);
```

Comment:
I see you used the `cors` middleware for the each route.
I would put it as one of first middlewares.
The similar with `checkToken`. Instead of putting it for the each route, put it into the "root" middleware.
e.g.
```
app.use(cors()); // <--cors
app.use(express.json());
app.use("/api/users", checkToken, usersRouter); // <-- checkToken
app.use("/api/groups", checkToken, groupsRouter); // <-- checkToken
app.use("/api/login", authRouter);
app.use(notFoundHandler);
app.use(errorHandler);
```